Makefile

```diff
@@ -156,9 +156,9 @@ e2e-coverage:
 kind-load: $(KIND) #EXHELP Loads the currently constructed image onto the cluster.
 ifeq ($(CONTAINER_RUNTIME),podman)
 	@echo "Using Podman"
-	podman save $(IMG) -o $(IMG).tar
-	$(KIND) load image-archive $(IMG).tar --name $(KIND_CLUSTER_NAME)
-	rm $(IMG).tar
+	$(CONTAINER_RUNTIME) save $(IMG) -o /tmp/$(shell echo $(IMG) | sed 's/[/:]/-/g').tar
+	$(KIND) load image-archive /tmp/$(shell echo $(IMG) | sed 's/[/:]/-/g').tar --name $(KIND_CLUSTER_NAME)
+	rm /tmp/$(shell echo $(IMG) | sed 's/[/:]/-/g').tar
```

Comment:
Would it make more sense to use `mktemp`?

Reply:
Just with the default randomly generated filename to simplify all of this?

Reply:
Or even just a shell variable for the filename.

Reply:
Ahh, yeah good idea lol